airbyte-workers/src/main/java/io/airbyte/workers/run/TemporalWorkerRunFactory.java

```diff
@@ -90,6 +90,7 @@ private static TemporalJobType toTemporalJobType(final ConfigType jobType) {
       case CHECK_CONNECTION_SOURCE, CHECK_CONNECTION_DESTINATION -> TemporalJobType.CHECK_CONNECTION;
       case DISCOVER_SCHEMA -> TemporalJobType.DISCOVER_SCHEMA;
       case SYNC, RESET_CONNECTION -> TemporalJobType.SYNC;
+      case CONNECTOR_BUILDER_READ -> TemporalJobType.CONNECTOR;
```

Comment:
thinking it's cleaner to have the new connector workers in a separate module to minimise code change complexity. Wdyt @girarda @jdpgrailsdev

Reply:
I agree.  If this won't be used by the workers, then it probably doesn't belong here.

Reply:
@davinchia I'm not sure I understand how the connector workers can be in another module. Are you suggesting we write them in a new module and add it as a dependency to airbyte-workers?

Reply:
Not exactly. I'm suggesting having a separate airbyte workers deploys in addition to the code being in a different module. In that way it's clear what is connector builder related and what is platform related.